app/src/main/java/org/simple/clinic/storage/Timestamps.kt

```diff
@@ -0,0 +1,42 @@
+package org.simple.clinic.storage
+
+import androidx.room.Embedded
+import org.simple.clinic.util.UtcClock
+import org.threeten.bp.Instant
+
+/**
+ * There are three timestamps that are always present in every single synced model:
+ *
+ * - `createdAt`: When the database record was created.
+ * - `updatedAt`: When the database record was updated.
+ * - `deletedAt`: When the database record was deleted.
+ *
+ * All timestamps are in UTC.
+ *
+ * This class is meant to encapsulate these timestamps as an [Embedded] model in the database
+ * entities.
+ **/
+data class Timestamps(
+    val createdAt: Instant,
+    val updatedAt: Instant,
+    val deletedAt: Instant?
+) {
+
+  companion object {
+
+    fun now(clock: UtcClock): Timestamps {
```

Comment:
This method should be called `create` since this will create timestamps for a new entry.

Reply:
True, will rename.

Reply:
Never mind, I saw that you addressed the feedback yourself. Thanks!